src/components/Markdown.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactMarkdown, { Components } from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+type MarkdownProps = {
+  text: string;
+};
+
+function CodeComponent({
+  inline,
+  className,
+  children
+}: {
+  inline?: boolean;
+  className?: string;
+  children?: React.ReactNode;
+}) {
+  if (inline) {
+    return <code className="rounded bg-gray-200 px-1 py-0.5">{children}</code>;
+  }
+
+  return (
+    <pre className="overflow-x-auto rounded bg-gray-900 p-2 text-white">
+      <code className={className}>{children}</code>
+    </pre>
+  );
+}
+
+const components: Components = {
+  code: CodeComponent as any
+};

```

Comment:
i think we need more components here since tailwind resets a bunch of styles. for example, see https://github.com/dstl-lab/dstl-onboarding-chat/blob/final/frontend/src/Markdown.tsx which has styling for `h1`, `h2`, `ul`, etc.

for example, the `h1` in the image below should be larger and bolded:

<img width="795" height="387" alt="Image" src="https://github.com/user-attachments/assets/248f3986-076a-4615-bdb5-5731d199d923" />